korangar/src/graphics/passes/directional_shadow/shader/entity_bindless.wgsl

```diff
@@ -47,26 +51,35 @@ fn vs_main(
 ) -> VertexOutput {
     let instance = instance_data[instance_index];
     let vertex = vertex_data(vertex_index);
+    let new_vertex = instance.affine * vec4<f32>(vertex.position, 1.0);
```

Comment:
We calculate the world coordinate by using both the affine and the world matrix directly after each other. This means, that we can pre-multiply the affine and world matrices on the CPU side! (and don't need to send both to the GPU)

Reply:
The problem is that for getting information about the depth and curvature is necessary to pass through the affine matrix.

Reply:
Okay, I see. Btw: the variable "world_position" is never used, so you can remove it.
Also: please find a better names for "new_vertex" and "affine". I think the annimation is "morphing" the base entity part, correct? Maybe "morphed_vertex" and "morph" are better names?

Reply:
Morph is a difficult word, I changed to frame_part_vertex and frame_part_transform.

Reply:
The new name is good!